testing/trino-product-tests/src/main/java/io/trino/tests/product/iceberg/TestIcebergSparkCompatibility.java

```diff
@@ -1746,6 +1746,56 @@ public void testTrinoReadsSparkRowLevelDeletes(StorageFormat tableStorageFormat,
         onSpark().executeQuery("DROP TABLE " + sparkTableName);
     }
 
+    @Test(groups = {ICEBERG, PROFILE_SPECIFIC_TESTS}, dataProvider = "tableFormatWithDeleteFormat")
+    public void testTrinoReadsSparkRowLevelDeletesWithAggregatePushdown(StorageFormat tableStorageFormat, StorageFormat deleteFileStorageFormat)
```

Comment:
What does this test add with respect to other tests?

Does it need to be a Product test? In particular, do we need Spark to create row-level delete files?

Reply:
To check regression the row level delete while the aggregate push down is enabled.

We can add to the product tests as well, if we are testing the row level deletes.